demand-capacity-mgmt-backend/src/main/resources/application.yml

```diff
@@ -36,17 +47,28 @@ spring:
 
   flyway:
     enabled: true
-    url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:5432}/${DCM_DATASOURCE_NAME:dcm}
+    url: jdbc:postgresql://${DCM_DATASOURCE_HOST:localhost:15432}/${DCM_DATASOURCE_NAME:dcm}
```

Comment:
Why did you have to change port. Even docker-compose you added is not using these ports. Local development install is also recommending these standard ports

Reply:
Fixed with https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/72/commits/b3a7dbc994b6739de126ade88eab1e05652201ba